docs/public/SUMMARY.md

```diff
@@ -30,6 +34,10 @@
 - [Specifications](specifications/README.md)
     - [Test Types](specifications/test-types/README.md)
         - [Undelegated Test](specifications/test-types/undelegated-test.md)
+    - [Test Zones](specifications/test-zones/README.md)
+        - [DNSSEC-TP](specifications/test-zones/DNSSEC-TP)
+        - [Nameserver-TP](specifications/test-zones/Nameserver-TP)
+        - [Zone-TP](specifications/test-zones/Zone-TP)
```

Comment:
Such entries will prevent building the documentation. They should link to a document not a folder.

Reply:
Will fix by including READMEs.